packages/resolve-scripts/src/template/client/reducers/todos.js

```diff
@@ -1,5 +1,5 @@
 import { createReducer } from 'resolve-redux';
-import readModel from '../../common/read-model';
+import readModel from '../../common/read-models';
```

Comment:
It's not enought to fix, due `read-models` directory index contains export for array of read models, and below only one read model is used. To minimize changes, it's better to use following construction: `import readModel from '../../common/read-models/default`